### DISCUSSION TITLE
Add feature which enables you to change the initial scroll position

### DISCUSSION THREAD

**Discussion by BlazeCodeDev** [patrykandpatrick/vico#303] [Questions]
I need the initial scroll position to be on the far right side of the chart, not on the far left side.

Is there already a function to accomplish this, if so; How does it work?

**Comment by patrickmichalik** [comment]
Hello! `ChartScrollSpec` takes an instance of `InitialScroll`. Passing in `InitialScroll.End` should produce the desired result.
